### ISSUE TITLE
A confusion about code

### ISSUE THREAD

**Issue by NzzzzN** [broliang/HRGAT#2]
Thanks for your amazing work! And I have a confusion about code.
![image](https://github.com/broliang/HRGAT/assets/43903956/0223545e-0a61-439f-b154-ea0327744fa0)
in class RGATConv def reduce_func
h = self.drop(torch.sum(alpha * nodes.mailbox['msg'], dim=1)) / 3
why divide by 3?
